### ISSUE TITLE
[Windows Terminal>Find]: Screen reader does not announce result position while navigating 'Find' suggestions.

### ISSUE THREAD

**Issue by Saurabh220304** [microsoft/terminal#19691]
### Windows Terminal version

1.25.3571.0

### Windows build number

Windows 11 Version 26H1 (OS Build 28000.1340)

### Steps to reproduce

**Repro Steps:**
1.Launch the Window terminal app.
2.Press 'Ctrl+Shift+F' keys to open the find dialog.
3.Find dialog will open.
4.Now turn on the narrator using 'CTRL+Win+Enter' keys.
5.Type any keyword like 'a' and press enter to search.
4.Now navigate to the 'Moves up/Down' button and press enter.
5.Observe the issue here.

**User Experience:**
Screen reader users cannot determine: 'How many search results are available, Their current position among the results'.
This violates accessibility expectations for feedback and navigation context.

**WCAG Reference Link:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

**Attachements:**
https://microsoftapc-my.sharepoint.com/:u:/g/personal/v-heparihar_microsoft_com/IQB_Klxg8ALBT7dUFH7Q2caqAUkNj7nzR7k1BBzoivpTXWQ?e=glq7zn

### Expected Behavior

>Screen reader should announce the position information for each result, for example:
“Result 1 of 9”
“Result 2 of 9”
“Result 3 of 9”, etc.

>This announcement should update dynamically as the user moves through the results.

### Actual Behavior

When using the Find functionality and multiple suggestions/results are available, the screen reader does not announce the position of the current result within the total number of results. Instead, it only announces the presence of 'Move Up and Move Down' button On/Off information.
This behavior makes it difficult for screen reader users to understand how many matches exist and which result they are currently on.

**Comment by carlos-zamora** [comment]
Looks like we may be able to set `SizeOfSet` and `PositionInSet` based on the calculated values. Worth a shot.
